src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -95,6 +96,9 @@ const SpendingByCFDAContainer = ({ onRedirectModalClick }) => {
                 order
             }
         };
+        if (activeTab !== 'all') {
+            params.filter.award_type_codes = awardTypeGroups[activeTab];
+        }
         const request = fetchSpendingByCfda(params);
```

Comment:
What about the cancel logic? Should we have that here? Like checking to see if request exists before we make a new one?